.buildkite/dra_pipeline.yml

```diff
@@ -4,8 +4,12 @@ steps:
   - label: ":pipeline: Generate steps"
     command: |
       set -euo pipefail
-
-      echo "--- Building [${WORKFLOW_TYPE}] artifacts"
+      
+      echo "--- Building [$${WORKFLOW_TYPE}] artifacts"
```

Comment:
this is not a typo, it was a bug :) double `$` are required other the variable is interpolated by BK.